### PR TITLE
Rewrite IDEA.md

### PR DESCRIPTION
**Pull request by Smith-Cruise**
The original tutorial could not be run locally, so it has been rewritten.

### CODE REVIEW

File: Contributors/guide/IDEA.md [hunk 1]
```diff
@@ -1,100 +1,141 @@
...
+## Requirements
 
-#### Mac
+**Thrift 0.13**
 
-On Mac you can install it with the following steps.
+If you installed Thrift directly with brew, you will find that there is no version 0.13, you can use the following command:
 
-```
+```bash
```
Comment (stephen-shelby) [inline Contributors/guide/IDEA.md:12]: Do we need to provide some compiled thrift bins.

File: Contributors/guide/IDEA.md [hunk 2]
```diff
@@ -1,100 +1,141 @@
...
+Even if you are not using macOS, you can also refer to this tutorial.
 
-Now, StarRocks needs thrift 0.13.x. So you need to install thrift-0.13.0 first.
+## Requirements
 
-#### Mac
+**Thrift 0.13**
 
-On Mac you can install it with the following steps.
+If you installed Thrift directly with brew, you will find that there is no version 0.13, you can use the following command:
```
Comment (stephen-shelby) [inline Contributors/guide/IDEA.md:10]: Thrift -> thrift ?

File: Contributors/guide/IDEA.md [hunk 3]
```diff
@@ -1,100 +1,141 @@
...
 brew extract --version='0.13.0' thrift $USER/local-tap
 brew install thrift@0.13.0
 ```
-After installing thrift successfully, you can check by executing following command in command-line.
 
-```
+After installing Thrift successfully, you can check by executing following command:
+
+```bash
 $ thrift -version
```
Comment (stephen-shelby) [inline Contributors/guide/IDEA.md:21]: I think the fastest way to compile successfully on idea is to modify https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/pom.xml#L65

File: Contributors/guide/IDEA.md (hunk 1 above)
Comment (imay) [reply Contributors/guide/IDEA.md:12]: No brew can help us install thrift 0.13

**Pull request merged**